action-scripts/build-config.js

```diff
@@ -1,8 +1,8 @@
 const fs = require("fs");
 
-const buildConfig = async ({
-  outputFolder = "build",
+const buildConfig = ({
   transformedTokenPath,
+  outputFolder = "build",
```

Comment:
Noticed that `outputFolder` and `transformedTokenPath` were in reverse order from where they were called from `buildConfig()` in `main.js`.